CONTRIBUTING.md

```diff
@@ -1,26 +1,26 @@
 ## Contributing to the C++ Core Guidelines
 
->"Within C++ is a smaller, simpler, safer language struggling to get out." 
->-- <cite>Bjarne Stroustrup</cite>
+>“Within C++ is a smaller, simpler, safer language struggling to get out.”<br />
```

Comment:
You should be able to add spaces to the end of the line to add a line-break using markdown, not html